app/src/main/java/ch/epfl/cs311/wanderwave/viewmodel/MapViewModel.kt

```diff
@@ -35,11 +28,13 @@ constructor(private val repository: BeaconRepositoryImpl, val locationSource: Lo
   }
 
   private fun observeBeacons() {
-    CoroutineScope(Dispatchers.IO).launch {
-      repository.getAll().collect { beacons ->
-        _uiState.value = BeaconListUiState(beacons = beacons, loading = false)
-      }
-    }
+    // CoroutineScope(Dispatchers.IO).launch {
+    //   repository.getAll().collect { beacons ->
+    //     _uiState.value = BeaconListUiState(beacons = beacons, loading = false)
+    //   }
+    // }
+
+    // TODO : Implement the repository with the local database
```

Comment:
By 'implement the repository', you mean creating the repository implementation and using it to get the beacons?

Reply:
Sorry this was a message for the past me, I should delete it, at the time I didn't know how to deal with implementing the firebase local storage, so I just wrote this to remind me to deal with this when I solved the problem. Which I didn't  implement, thanks for pointing this out.